protocol/protobuf/pairing.proto

```diff
@@ -119,3 +119,13 @@ message SyncClearHistory {
   string chat_id = 1;
   uint64 cleared_at = 2;
 }
+
+message SyncProfilePicture {
+  string key_uid = 1;
+  string name = 2;
+  bytes  payload = 3;
+	uint32 width = 4;
+	uint32 height = 5;      
+	uint32 file_size = 6;
+	uint32 resize_target = 7;
+}
```

Comment:
You could add the sync clock to the `SyncProfilePicture` protobuf and then batch all your `SyncProfilePicture`s together with something like:

```proto
message SyncProfilePicture {
  string key_uid = 1;
  string name = 2;
  bytes payload = 3;
  uint32 width = 4;
  uint32 height = 5;      
  uint32 file_size = 6;
  uint32 resize_target = 7;
  uint64 clock = 8;
}

message SyncProfilePictures {
  repeated SyncProfilePicture pictures = 1;
}
```

Reply:
Done.